.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Release-plz
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release-plz:
+    name: Release-plz
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install Rust (rustup)
+        run: rustup update nightly --no-self-update && rustup default nightly
+      - name: Run release-plz
+        uses: MarcoIeni/release-plz-action@v0.5
```

Comment:
this seems to now be at https://github.com/release-plz/action

Reply:
Indeed - I updated that, and also split the jobs as seems to be recommended now from https://release-plz.ieni.dev/docs/github/quickstart#3-setup-the-workflow